src/cleanlab_tlm/utils/chat.py

```diff
@@ -141,7 +187,7 @@ def _form_prompt_responses_api(messages: list[dict[str, Any]], tools: Optional[l
                 output += f"<tool_response>\n{json.dumps(tool_response, indent=2)}\n</tool_response>\n\n"
         else:
             role = msg.get("name", msg["role"])
-            if role == "system":
+            if role in SYSTEM_ROLES:
                 prefix = "System: "
             elif role == "user":
                 prefix = "User: "
```

Comment:
you want to define string variables in the file, outside of the functions:

```
SYSTEM_PREFIX = "System: "
USER_PREFIX = "User: "
...
```
 same for any other hardcoded string that is being defined within a function.  Within the functions you should only refer to the variables, not the hardcoded strings themselves.

Reply:
things like this should also be defined as variable outside the function:

```
TOOL_DEFINITIONS_PREFIX = 
(
        "You are a function calling AI model. You are provided with function signatures within <tools> </tools> XML tags. "
        "You may call one or more functions to assist with the user query. If available tools are not relevant in assisting "
        "with user query, just respond in natural conversational language. Don't make assumptions about what values to plug "
        "into functions. After calling & executing the functions, you will be provided with function results within "
        "<tool_response> </tool_response> XML tags.\n\n"
        "<tools>\n"
    )
```